src/libmongoc/src/mongoc/mongoc-apm.h

```diff
@@ -104,6 +104,11 @@ mongoc_apm_command_started_get_service_id (
    const mongoc_apm_command_started_t *event);
 MONGOC_EXPORT (int32_t)
 mongoc_apm_command_started_get_server_connection_id (
+   const mongoc_apm_command_started_t *event)
+   BSON_GNUC_DEPRECATED_FOR (
```

Comment:
Can we add a `MONGOC_WARNING` to the deprecated `mongoc_apm_command_(started|succeeded|failed)_get_server_connection_id` functions just prior to returning `MONGOC_NO_SERVER_CONNECTION_ID;` as a runtime indicator for when the deprecated function is returning `-1` due to out-of-range rather than an actual "no serverConnectionId" error?

Reply:
That seems helpful. Done.